test/subset/meson.build

```diff
@@ -0,0 +1,23 @@
+tests = [
+  ['basics.tests'],
+  ['full-font.tests'],
+  ['japanese.tests'],
+]
+
+# The test tries to run which unconditionally, windows doesn't agree
```

Comment:
I don’t think this is true anymore, use of `which` command drooped in f57804a8a596e88843ddc8b88afac7526349b89b.

Reply:
Fixed.